app/assets/javascripts/short-form/templates/c4-household-public-housing.html.slim

```diff
@@ -1,14 +1,22 @@
 .app-card.form-card
   short-form-header
-    h2.app-card_question
-      | {{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE' | translate: youOrHouseholdTranslateVariable() }}
+    h2.app-card_question ng-if="householdMembers.length == 0"
+      | {{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE_YOU' | translate }}
+    h2.app-card_question ng-if="householdMembers.length > 0"
+      | {{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE_HOUSEHOLD' | translate }}
     p.form-note.margin-bottom
       | {{'C4_HOUSEHOLD_PUBLIC_HOUSING.P1' | translate}}
 
   .app-inner.inset
     .form-group ng-class="{ error: inputInvalid('STUB_householdPublicHousing') }"
       p.form-note.margin-bottom translate="LABEL.PLEASE_SELECT_ONE"
-      .radio-group.padding-top--half role="radiogroup" aria-label="{{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE' | translate}}"
+      .radio-group.padding-top--half role="radiogroup" aria-label="{{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE_YOU' | translate}}" ng-if="householdMembers.length == 0"
+        radio-block-item value='Yes' user='application' name='STUB_householdPublicHousing' ng-required='true' on-change="publicHousingYes"
```

Comment:
@kaptron yeah I think that's why we did a function to render the right words.. but yeah! would it still serve the right purpose of having `aria-labels` if we did it this way:

```
h2.app-card_question ng-if="householdMembers.length == 0" aria-label="{{'C6A_HOUSEHOLD_RESERVED_UNITS_VETERAN.TITLE_YOU' | translate}}"
  | {{ 'C6A_HOUSEHOLD_RESERVED_UNITS_VETERAN.TITLE_YOU' | translate }}
h2.app-card_question ng-if="householdMembers.length > 0" aria-label="{{'C6A_HOUSEHOLD_RESERVED_UNITS_VETERAN.TITLE_HOUSEHOLD' | translate}}" 
  | {{ 'C6A_HOUSEHOLD_RESERVED_UNITS_VETERAN.TITLE_HOUSEHOLD' | translate }}
```

that will remove the radio buttons.